package.json

```diff
@@ -51,7 +51,7 @@
     "eslint-plugin-ember": "^5.0.0",
     "ethers": "3.0.15",
     "kosmos-schemas": "^1.1.2",
-    "kredits-contracts": "github:67P/kredits-contracts",
+    "kredits-contracts": "^3.x",
```

Comment:
I think the `^` and `.x` are doing the same thing. So we should use either one of those. Personally, I like `3.x` a bit better as it's more expressive.

Reply:
Weird: I already changed this and it's `3.x` in master, so I'm not sure what's going on here.

Reply:
#metoo